docs/MEMORY-SETTINGS.md

```diff
@@ -220,6 +220,36 @@ Changing these parameters is recommended as a last resort, for example to debug
 an OOM kill due to a bloated `zedbox` process. Before changing the values,
 please read the [documentation](https://tip.golang.org/doc/gc-guide) carefully.
 
+## Forced execution of the Golang runtime garbage collector settings
+
+Setting a soft limit for Golang garbage collector may not be enough:
+the whole system may experience memory pressure due to other
```

Comment:
The proposed approach does not handle the global memory pressure events. Only events correspond to reaching the limit in /eve or child cgroups.

Reply:
Right, changed to the root mem cg.

Reply:
Just for the clarity. One of the meaning of CONSTRAINT_NONE is that application does not belong to any of the non-root cgroups (see the `constained_alloc()` function), basically `memcg` pointer for the `oom_control` is null. But this does mean that `vmpressure` events won't be generated (see the `memcg_to_vmpressure()` function and their callers, what exactly happens if `memcg` is null) - events  all land in the root memcg. For this particular case it is enough to listen to the root memcg (I've just changed that) and you get those events not only /eve group related. To reproduce is easy - default shell does not belong to any of the cgroups, any application started from shell will be killed once all memory on the host is consumed, and there will be a single event in the `logread` about memory pressure and attempt to call the GC.